neo.UnitTests/UT_Transaction.cs

```diff
@@ -55,16 +65,79 @@ public void Size_Get()
             uut.Script = TestUtils.GetByteArray(32, 0x42);
             uut.Sender = UInt160.Zero;
             uut.Attributes = new TransactionAttribute[0];
-            uut.Witness = new Witness
+            uut.Witnesses = new Witness[]{ new Witness
             {
                 InvocationScript = new byte[0],
                 VerificationScript = new byte[0]
-            };
+            } };
 
             uut.Version.Should().Be(0);
             uut.Script.Length.Should().Be(32);
             uut.Script.GetVarSize().Should().Be(33);
-            uut.Size.Should().Be(81);
+            uut.Size.Should().Be(82);
+        }
+
+        [TestMethod]
+        public void FeeIsSignatureContract()
+        {
+            var wallet = new NEP6Wallet("unit-test.json");
```

Comment:
Shargon, a better approach to this is here: https://github.com/neo-project/neo/pull/847

Reply:
I think create a temp wallet is okay.